src/mlpack/methods/ann/layer/leaky_relu_impl.hpp

```diff
@@ -42,7 +42,8 @@ void LeakyReLU<InputDataType, OutputDataType>::Backward(
 {
   DataType derivative;
   derivative.set_size(arma::size(input));
-  for (size_t i = 0; i < input.n_elem; i++)
+  #pragma omp parallel for
+  for (omp_size_t i = 0; i < input.n_elem; i++)
```

Comment:
Again, I think this loop contains a pretty simple operation. I guess in this case the performance depends on the memory clock speed rather than on the CPU.

Reply:
Again, I think it would depend on the size of the input, right?